includes/core-changes/interop/5.0/casting-rcw-to-inspectable-interface-throws-exception.md

```diff
@@ -0,0 +1,53 @@
+### Casting RCW to an `InterfaceIsIInspectable` interface throws PlatformNotSupportedException
+
+Casting a runtime callable wrapper (RCW) to an interface marked as <xref:System.Runtime.InteropServices.ComInterfaceType.InterfaceIsIInspectable> now throws a <xref:System.PlatformNotSupportedException>.
+
+#### Version introduced
+
+5.0 RC2
+
+#### Change description
+
+In previous .NET versions, you can succesfully case an RCW to an interface marked as <xref:System.Runtime.InteropServices.ComInterfaceType.InterfaceIsIInspectable>. However, you might get access violations when you execute methods on the interface.
```

Comment:
This is a breaking change from previous .NET versions, e.g. 3.1, not just from a previous .NET 5 preview version, correct?

Reply:
The access violations could only happen on previous .NET 5 previews.

Reply:
On .NET 3.1 and earlier, this works as expected.